podpac/core/algorithm/algorithm.py

```diff
@@ -176,7 +177,7 @@ def algorithm(self):
        
         c = ec[coord_name]
         data = c.coordinates
-        coords = Coordinate(order=[coord_name], **{coord_name: c})
+        coords = Coordinate(OrderedDict(**{coord_name:c}))
```

Comment:
Why not just 
`OrderedDict((coord_name, c))` ?

Reply:
Yeah, that's better. I think you would need `OrderedDict([(coord_name, c)])`.